Dockerfile

```diff
@@ -13,7 +13,7 @@ ADD requirements /pip/requirements
 
 # Setting cwd to /pip ensures egg-links for git installed deps are created in /pip/src
 WORKDIR /pip
-RUN pip install -b /pip/build --download-cache /pip/cache --no-deps -r /pip/requirements/docker.txt
+RUN pip install -b /pip/build --download-cache /pip/cache --no-deps -r /pip/requirements/docker.txt --find-links https://pyrepo.addons.mozilla.org/
```

Comment:
Worked for me. Also a problem since pyjwt 1.4.2 was removed from pypi, but not updated in requirements yet, meaning webpay wouldn't build.